.gitattributes

```diff
@@ -0,0 +1 @@
+*  text=auto eol=lf
```

Comment:
Curious what this is for, I'm unfamiliar

Reply:
[ensures that windows' users don't accidentally check in the wrong line endings](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#per-repository-settings).

Reply:
Ah great! Thanks for explaining 🙂